README.md

```diff
@@ -24,3 +26,11 @@ Flags:
 Global Flags:
       --config string   config file (default is $HOME/.test-brain.yaml)
 ```
+
+## Marking tests as skipped
+
+Sometimes a test may be marked as skipped, which indicates it didn't fail neither succeeded. It may
```

Comment:
-it didn't fail neither succeeded.
+it neither failed nor succeeded.

(neither A nor B `<=>` not A and not B)